stingray/gti.py

```diff
@@ -263,6 +264,7 @@ def create_gti_mask(time, gtis, safe_interval=0, min_length=0,
         create_gti_mask_jit((time - time[0]).astype(np.float64),
                             (gtis_to_mask - time[0]).astype(np.float64),
                             mask, gti_mask=gti_mask, min_length=min_length)
+
```

Comment:
I didn't want to modify the original gtis, because that is a possibly buggy side effect. I'd say that generally gtis are a tiny amount of memory. Even thousands of GTIs take sub-seconds to process.

Reply:
👍